dev/com.ibm.ws.cdi.interfaces/src/com/ibm/ws/cdi/CDIService.java

```diff
@@ -75,9 +78,20 @@ public interface CDIService {
     /**
      * Returns all interceptor bindings which apply to the current invocation or lifecycle event.
      *
-     * @return a set of interceptor bindings which apply to the current invocation or lifecycle event. This will include all interceptor bindings that apply, not just those that were used to bind the current interceptor.
+     * @return a set of interceptor bindings which apply to the current invocation or lifecycle event. This will include all interceptor bindings that apply, not just those that
+     *         were used to bind the current interceptor.
      * @throws IllegalArgumentException if InvocationContext is not an instance of org.jboss.weld.interceptor.proxy.AbstractInvocationContext;
      */
     public Set<Annotation> getInterceptorBindingsFromInvocationContext(InvocationContext ic) throws IllegalArgumentException;
 
+    /**
+     * Gets the J2EEName of the Web or EJB module containing the given class (if any)
+     * <p>
+     * Result is non-deterministic if the class appears in two modules (e.g. same jar is referenced on the Manifest-Classpath of two wars)
```

Comment:
In this case, would both classes be loaded by the same classloader?

Reply:
fixed